steps/wetransfer/v0.0.3/step.yml

```diff
@@ -0,0 +1,61 @@
+title: wetransfer
+summary: |
+  Upload files to wetransfer and share a short url to download
+description: |
+  This step send files to wetransfer. Wetransfer give you a short url to download files. Limit 2GO.
+website: https://github.com/kawaiseb/bitrise-step-wetransfer
+source_code_url: https://github.com/kawaiseb/bitrise-step-wetransfer
+support_url: https://github.com/kawaiseb/bitrise-step-wetransfer/issues
+published_at: 2018-02-08T19:10:33.180517+01:00
+source:
+  git: https://github.com/kawaiseb/bitrise-step-wetransfer.git
+  commit: e5403903fc40ab5a919f4cdb110eca8deae0dd8a
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+type_tags:
+- utility
+toolkit:
+  bash:
+    entry_file: step.sh
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- WTU_MAILSENDER: $WTU_MAILSENDER
+  opts:
+    is_expand: false
```

Comment:
if you set is_expand: false, the step will receive: `$WTU_MAILSENDER`, instead of the value of the env: WTU_MAILSENDER.
i would recommend to use is_expand: true.

Reply:
Ok thanks.